web/src/test/java/se/inera/intyg/intygsadmin/web/controller/StatusControllerTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygsadmin.web.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.intygsadmin.web.controller.dto.SendStatusResponseDTO;
+import se.inera.intyg.intygsadmin.web.integration.dto.SendStatusForCareGiverIntegrationRequestDTO;
+import se.inera.intyg.intygsadmin.web.integration.dto.SendStatusForCertificatesIntegrationRequestDTO;
+import se.inera.intyg.intygsadmin.web.integration.dto.SendStatusForTimePeriodIntegrationRequestDTO;
+import se.inera.intyg.intygsadmin.web.integration.dto.SendStatusForUnitsIntegrationRequestDTO;
+import se.inera.intyg.intygsadmin.web.service.status.NotificationStatusEnum;
+import se.inera.intyg.intygsadmin.web.service.status.SendStatusForCareGiverService;
+import se.inera.intyg.intygsadmin.web.service.status.SendStatusForCertificatesService;
+import se.inera.intyg.intygsadmin.web.service.status.SendStatusForTimePeriodService;
+import se.inera.intyg.intygsadmin.web.service.status.SendStatusForUnitsService;
+import se.inera.intyg.intygsadmin.web.service.status.SendStatusService;
+
+@ExtendWith(MockitoExtension.class)
+class StatusControllerTest {
+
+    public static final String STATUS_ID = "statusId";
+    @Mock
+    private SendStatusService sendStatusService;
+
+    @Mock
+    private SendStatusForCertificatesService sendStatusForCertificatesService;
+
+    @Mock
+    SendStatusForUnitsService sendStatusForUnitsService;
+
+    @Mock
+    private SendStatusForCareGiverService sendStatusForCareGiverService;
+
+    @Mock
+    private SendStatusForTimePeriodService sendStatusForTimePeriodService;
+
+    @InjectMocks
+    private StatusController statusController;
+
+    @Test
+    void sendStatus() {
```

Comment:
Not necessary to change but for the future: I know a lot prefer to write tests like "should..." or "shall..."